picocrt/meson.build

```diff
@@ -102,6 +109,13 @@ foreach target : targets
 	       _crt.extract_objects(src_picocrt)
 	      )
 
+  static_library('crt0hosted',
```

Comment:
You'll need to deal with multilib just like the crt0*.o files. That's probably most easily done by removing the .o from the `crt_name`, `crt_hosted_name` etc values and appending the `.o` back on for the executable targets and using it plain for the static library targets.

Reply:
Thanks for the suggestion. Also this ensures the library and object file names are exactly consistent with each other, which in my previous approach they accidentally weren't!